app/models/curation_concerns/_form_additional_information.html.erb

```diff
@@ -0,0 +1,12 @@
+<fieldset class="optional prompt">
+  <legend>Additional Information</legend>
+  <%= f.input :subject,                as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :publisher,              as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :date %>
+  <%= f.input :source,                 as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :language,               as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :isbn,                   as: :multi_value, input_html: { class: 'form-control' }, label: 'ISBN' %>
```

Comment:
I'm not sure why this isn't showing up in the diff, but they were removed in commit 77171e6